selfdrive/car/toyota/carcontroller.py

```diff
@@ -63,6 +65,9 @@ def update(self, CC, CS):
     if not CC.enabled and CS.pcm_acc_status:
       pcm_cancel_cmd = 1
 
+    if not pcm_cancel_cmd and self.prev_pcm_cancel:
+      self.disengage_frame = self.frame
```

Comment:
TODO: see how quickly stock cruise cancels, we might be able to get away with no prev_pcm_cancel intermediate variable if we can start our sequence as soon as we start cancelling